.github/workflows/cd.yaml

```
@@ -45,9 +45,9 @@ jobs:
         with:
           fetch-depth: 0
       - name: Set up JDK 8
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3.0.0
```

Switch to declare only the major version because we have no facility to evaluate new versions in a CI job.
```suggestion
        uses: actions/setup-java@v3
```